pkg/network/node/node.go

```diff
@@ -439,7 +439,7 @@ func (node *OsdnNode) killFailedPods(failed map[string]*kruntimeapi.PodSandbox)
 // See https://github.com/containernetworking/cni/issues/89
 func (node *OsdnNode) UpdatePod(pod corev1.Pod) error {
 	filter := &kruntimeapi.PodSandboxFilter{
-		LabelSelector: map[string]string{ktypes.KubernetesPodUIDLabel: string(pod.UID)},
+		LabelSelector: map[string]string{ktypes.KubernetesPodUIDLabel: string(pod.UID)}, //this const was removed from ktypes and am unsure what to replace it with
```

Comment:
Possibly something in `kruntimeapi` since this is a `kruntimeapi.PodSandboxFilter`?

One trick that is sometimes useful is to check out a copy of the N-1 version of kubernetes/kubernetes, and find places using that symbol, and then check out the current version of kubernetes, and see what that code is doing now. (_Hopefully_ there's just a different define, but possibly there was some sort of larger API change...)

Reply:
yeah was definetly part of a larger api change as the SwapBehavior types (where that value was a part of) no longer exists (seems like what happened was the label keys that were being used have replaced the old swap behavior types) 

we should be able to replace this with the string value of the old constant right?

Reply:
Well, the problem is that if the behavior on the other side of the API has changed, then the old constant value may no longer have the effect we wanted it to have.

What we're trying to do here, overall, is get the sandboxID of a particular pod, which we are currently identifying by UID. If we can't fix the `PodSandboxFilter` to do what we want, you could also just modify `getPodSandboxID` to not use a filter, and scan the returned list of pods itself. (This function is the only caller of `getPodSandboxID`.)